index.html

```diff
@@ -250,32 +250,27 @@ <h1 id="header-text">
                 <img src="./images/Seperator-line.svg" alt="seperator"></li>
               </ul>
             </div>
-
         </div>
-         <div id="contact-me">
+        <div id="contact-me">
+          <img src="./images/Ellipse 3.png" alt="eclipse-3" id="ellipse-3">
             <h2 id="contact-me-h2">Contact me</h2>
             <p id="contact-me-text">If you have and application you <br> 
                 are interested in developing, a <br> 
                 feature that you need built or a <br>
                 project that needs coding. <br>
                 I'd love to help with it.
             </p>
-
-            <form action="">
-              <input type="text"  required placeholder="Lorem Ipsum" class="name-email"> <br>
-              <input type="email" required placeholder="loremipsum@gmail.com" class="name-email" id="email"> <br>
-              <input type="text" required placeholder="Write your message here..." id="txt-message"> <br>   
-              <button class="button btn-border" id="in-touch-btn">Get in touch</button>
+            <!-- The Pair Programing code starts here below -->
+            <form action="https://formspree.io/f/xjvdaonp" method="post" id="contact-form">
+              <input type="text"  required placeholder="Lorem Ipsum" class="name-email" maxlength="30" id="user-name" name="user_name">
+              <input type="email" name="email" required placeholder="loremipsum@gmail.com" class="name-email" id="email">
```

Comment:
Kindly consider changing your `placeholder` text. Your placeholder text should be short and descriptive to tell the user what to input in the form field. This will create a great user experience.